tf2_geometry_msgs/CMakeLists.txt

```diff
@@ -20,7 +20,8 @@ ament_python_install_package(${PROJECT_NAME}
 add_library(${PROJECT_NAME} INTERFACE)
 target_include_directories(${PROJECT_NAME} INTERFACE
   "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-  "$<INSTALL_INTERFACE:include>")
+  "$<INSTALL_INTERFACE:include>"
+  ${orocos_kdl_INCLUDE_DIRS})
```

Comment:
I would hope linking to the `orocos-kdl` target would be enough to get the include directories, so this line could be removed

Reply:
It is not enough, unfortunately. Upstream does not appear to be exporting the dependency on Eigen:

```
--- stderr: tf2_kdl                                                            
In file included from /home/jacob/ws/ros/orocos_upstream/install/orocos_kdl_vendor/include/kdl/jntarray.hpp:26,
                 from /home/jacob/ws/ros/orocos_upstream/install/orocos_kdl_vendor/include/kdl/frames_io.hpp:81,
                 from /home/jacob/ws/ros/orocos_upstream/src/ros2/geometry2/tf2_kdl/test/test_tf2_kdl.cpp:42:
/home/jacob/ws/ros/orocos_upstream/install/orocos_kdl_vendor/include/kdl/jacobian.hpp:26:10: fatal error: Eigen/Core: No such file or directory
   26 | #include <Eigen/Core>
      |          ^~~~~~~~~~~~
compilation terminated.
```

 I presume any fix upstream is unlikely to make it into Jammy though.